CromwellPostgresLocalMinikube/templates/cromwell.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
         - name: cromwell-container
-          image: broadinstitute/cromwell:66-5f8a277
+          image: broadinstitute/cromwell:71-c778c33
```

Comment:
Whether or not you change this one doesn't really matter…this isn't our active helm chart.

Reply:
Should I revert this change for the sake of consistency?

Reply:
I guess it might as well be updated.